lib/exposes.js

```diff
@@ -378,6 +378,14 @@ class Climate extends Base {
         return this;
     }
 
+    withSetpointLimit(property, min, max, step, access=a.ALL) {
```

Comment:
Please expose these a separate numeric exposes (do not integrate into the `Climate` expose)

Reply:
Hello @Koenkk,
i've all update. Thanks you.

But for 
```
    withSetpointLimit(property, min, max, step, access=a.ALL) {
        assert(!this.endpoint, 'Cannot add feature after adding endpoint');
        assert(['min_heat_setpoint_limit', 'max_heat_setpoint_limit', 'min_cool_setpoint_limit', 'max_cool_setpoint_limit'].includes(property));
        this.features.push(new Numeric(property, access)
            .withValueMin(min).withValueMax(max).withValueStep(step).withUnit('Â°C').withDescription('Temperature setpoint limit'));
        return this;
    }
```
i think it's for climate, because  `assert(['min_heat_setpoint_limit', 'max_heat_setpoint_limit', 'min_cool_setpoint_limit', 'max_cool_setpoint_limit'].includes(property));`

It's ok for you ?

Reply:
I don't want to add more things to climate, only core functionality should be in there, rest are separate sensors.

Reply:
> I don't want to add more things to climate, only core functionality should be in there, rest are separate sensors.

ok :-) can you tell me how to do please...  in exposes file ? or other ?